shell/edit/logging.banzaicloud.io.output/index.vue

```diff
@@ -19,14 +20,44 @@ import YamlEditor, { EDITOR_MODES } from '@shell/components/YamlEditor';
 
 export default {
   components: {
-    Banner, CruResource, Labels, LabeledSelect, NameNsDescription, Tab, Tabbed, YamlEditor
+    Banner, CruResource, Labels, LabeledSelect, NameNsDescription, Tab, Tabbed, YamlEditor, Loading
   },
 
   mixins: [CreateEditView],
 
-  data() {
-    const schemas = this.$store.getters['cluster/all'](SCHEMA);
+  async fetch() {
+    this.resourceSchema = await this.$store.dispatch('cluster/find', { type: SCHEMA, id: LOGGING.OUTPUT });
```

Comment:
resourceSchema and schemaDefinition are only used in fetch now, so can be local `const`s